src/grpcbox_services_sup.erl

```diff
@@ -48,6 +48,23 @@ init([ServerOpts, GrpcOpts, ListenOpts, PoolOpts, TransportOpts, ServiceSupName]
 
     %% unique name for pool based on the ip and port it will listen on
     PoolName = pool_name(ListenOpts),
+    case {maps:get(accept_rate, ListenOpts, undefined),
```

Comment:
The max_connection settings is specified in the server opts, whilst the rate limiting here is in the listen opts.  Should we keep them in the same section ?
The listenopts are not threaded through as part of the acceptor state, hence why the max_connections is in server opts.  It also keeps it with the existing `max_concurrent_streams` settings

Reply:
moved all to ServerOpts